week 1 challenge

Amazingrace1 opened the pull request:
Amazingrace1 and codedentwickler

THIS IS A PULL REQUEST TEMPLATE! This helps you complete a pull request. Please follow the instructions below. 

## Description
week 1 problem set


**Please make sure your submission is in the correct directory**. In most cases, this is in submissions/YOUR-GITHUB-USERNAME/Week-Number/filename.

For example, if @codedentwickler is completing Week 1:
- I solved the Week 1 task.
- I create a new file inside submissions/codedentwickler/Week1/
- I name this file fizzbuzz.py, hence we have the fizzbuzz solution in submissions/codedentwickler/Week1/fizzbuzz.py

Amazingrace1 committed:
Merge pull request #1 from CottaCush/master

Amazingrace1 and codedentwickler

Adedipupo committed:
modified:   aboutme.md
	new file:   submissions/Amazingrace/Week_1_solution.py/fizzbuzz.php
	new file:   submissions/Amazingrace/aboutme.md

Amazingrace1 committed:
Merge pull request #2 from CottaCush/master

Amazingrace

Adedipupo committed:
Merge branch 'master' of https://github.com/Amazingrace1/Dufuna

codedentwickler commented on aboutme.md:
```diff
@@ -1,20 +1,19 @@
 
-I really cant choose anyone right now because that would be cheating, i love them all equally but if i was to choose, id go with 🦄, you just dont see that everyday 😆.
+🤨
 
```
Kindly delete this file and update your Pull request, it is a duplicate of what is in ` submissions/Amazingrace/aboutme.md`. Your aboutme.md should only be in your submission folder.

codedentwickler reviewed (changes requested):
Some changes to be made. We can merge once those changes have been made.

Adedipupo committed:
deleted:    submissions/Amazingrace/aboutme.md
	renamed:    submissions/Amazingrace/Week_1_solution.py/fizzbuzz.php -> submissions/Amazingrace1/Week_1_solution/fizzbuzz.php
	renamed:    aboutme.md -> submissions/Amazingrace1/aboutme.md
	deleted:    submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift
	deleted:    submissions/codedentwickler/Week1/Fizzbuzz.playground/contents.xcplayground
	deleted:    submissions/codedentwickler/aboutme.md

tegaphilip commented:
@Amazingrace1 looks like you deleted some files that should not be deleted. Check the changes you made again. You should not touch any file that is in someone else's submission's folder. After making changes, you can update your pull request.
